src/ResourceManager/DevTestLab/Commands.DevTestLab/AzureRM.DevTestLab.psd1

```diff
@@ -0,0 +1,88 @@
+#  
```

Comment:
Thanks. Fixed. I modified the psd1 file accordingly. I've now added dependency on AzureRM.Profile.